packages/netlify-cms-lib-util/src/APIUtils.ts

```diff
@@ -2,7 +2,7 @@ export const CMS_BRANCH_PREFIX = 'cms';
 export const DEFAULT_PR_BODY = 'Automatically generated by Netlify CMS';
 export const MERGE_COMMIT_MESSAGE = 'Automatically generated. Merged on Netlify CMS.';
 
-const DEFAULT_NETLIFY_CMS_LABEL_PREFIX = 'netlify-cms/';
+export const DEFAULT_NETLIFY_CMS_LABEL_PREFIX = 'netlify-cms/';
```

Comment:
Don't think we need to export this `const`

Reply:
I'm using this in the implementation file but if you prefer not exporting it, I can simply hard-code the string in the other file.

Reply:
See my other comments. I think once they are resolved we won't need to export it